### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Please review my login page

### CODE REVIEW

File: pages/stylesheets/login.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    height: 100%;
+    width: 100%;
+    background-image: linear-gradient(var(--shadow-color), var(--shadow-color)), url('../login-photo.jpg');
+    background-position: center;
+    background-size: cover;
+    position: absolute;
+}
+.form-box{
+    width: 380px;
+    height: 480px;
```
Comment (rohanmathur91) [inline pages/stylesheets/login.css]: use rem instead of pixels

File: pages/stylesheets/login.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    position: relative;
+    margin: 6% auto;
+    background:var(--bg-code);
+    padding: 5px;
+}
+.button-box{
+    width: 100px;
```
Comment (rohanmathur91) [inline pages/stylesheets/login.css]: same as above

**Review by rohanmathur91 — APPROVED** [review]
Added a few comments, the rest looks good to me

**Review by dev-enforced — APPROVED** [review]
The code looks good to me. Apply the changes as suggested by @rohanmathur91 and then you can merge it!! Good work Ranita on the design: https://deploy-preview-2--trekkart-store.netlify.app/pages/login.html

**Pull request merged**